src/mapml.js

```diff
@@ -1034,8 +1034,384 @@ M.QueryHandler = L.Handler.extend({
       }
     }
 });
+
+M.ContextMenu = L.Handler.extend({
+  _touchstart: L.Browser.msPointer ? 'MSPointerDown' : L.Browser.pointer ? 'pointerdown' : 'touchstart',
+
+  initialize: function (map) {
+    L.Handler.prototype.initialize.call(this, map);
+
+    //setting the items in the context menu and their callback functions
+    this._items = [
+      {
+        text:"Back (B)",
```

Comment:
These items should preferably use the [`<kbd>`](https://html.spec.whatwg.org/multipage/text-level-semantics.html#the-kbd-element) element to denote keyboard user input, for example:
```diff
-text:"Back (B)"
+text:"Back (<kbd>B</kbd>)"
```